source/features/clean-repo-sidebar.tsx

```diff
@@ -8,10 +8,43 @@ import features from '.';
 async function init(): Promise<void> {
 	document.body.classList.add('rgh-clean-repo-sidebar');
 
-	// Hide redundant "Releases" section
+	// Hide "Readme" link made unnecessary by toggle-files-button #3580
+	(await elementReady('.repository-content .BorderGrid-row:first-child .muted-link[href="#readme"]', {waitForChildren: false}))?.parentElement!.remove();
```

Comment:
There appears to be only one on the page:

```suggestion
	(await elementReady('.muted-link[href="#readme"]', {waitForChildren: false}))?.parentElement!.remove();
```